regen.ml

```diff
@@ -85,7 +85,7 @@ let generate_cas pem =
     String.concat ";\n" (List.map (fun ca -> Format.asprintf "{|%s|}" ca) cas)
   in
   let file = Format.sprintf "let certificates = [\n%s\n] " cas in
-  let oc = open_out "cas.ml" in
+  let oc = open_out "certificates.ml" in
```

Comment:
While testing regen locally, I found a bug where we are creating a new `cas.ml` file with the same content as `certificates.ml` file. If this is desired, I'll revert it back.

Reply:
This looks good, it was likely an oversight!